src/Utilities.js

```diff
@@ -16,3 +16,20 @@ export const resourceToName = (uri) => {
     return undefined
   }
 }
+
+export const isRepeatable = (repeatable) => {
```

Comment:
cool.  I think maybe we could name this "isTrue()" or "templateBoolean() and name the argument "value" instead of "repeatable".  I think that's worth doing for code clarity, but I won't hold up approval - you can do it or not before a merge.

Reply:
👍